### PR TITLE
run ETB on new TPS

### PR DESCRIPTION
**Pull request by mck1117**
Progress toward #1240

### CODE REVIEW

File: firmware/controllers/actuators/electronic_throttle.cpp [hunk 1]
```diff
@@ -109,6 +102,13 @@ static bool startupPositionError = false;
 
 #define STARTUP_NEUTRAL_POSITION_ERROR_THRESHOLD 5
 
+static SensorType indexToTpsSensor(size_t index) {
+	switch(index) {
```
Comment (rusefi) [inline firmware/controllers/actuators/electronic_throttle.cpp:106]: code style - space is missing?

Comment (mck1117) [reply firmware/controllers/actuators/electronic_throttle.cpp:106]: you mean between `switch` and `(index)`?

Comment (mck1117) [reply firmware/controllers/actuators/electronic_throttle.cpp:106]: lol why'd you merge before I could fix this 🤣

Comment (rusefi) [reply firmware/controllers/actuators/electronic_throttle.cpp:106]: yes before "switch" and "("

**Pull request merged**